community/mei-by-laws.md

```diff
@@ -73,41 +77,51 @@ The Board is self-organizing; that is, there will be no formal regulations on ho
 
 ### 5.1 Members
 
-The Board consists of nine (9) elected members plus one (1) member designated by each Host Institution (for details see 7 Host Institutions). All members of the Board (elected and designated) have the right to participate in decision-making processes established by the Board. Any member of the Board may resign from the Board on written notification to the administrative chair (see below).
-
-### 5.2 Meetings
+The Board consists of nine (9) elected members plus one (1) member designated by each Host Institution (for details see 7 Host Institutions). All members of the Board (elected and designated) have the right to participate in decision-making processes established by the Board.
 
-The Board holds regular, virtual or in-person meetings, e.g. approximately every three months. As schedules and funding permit, the Board meets in person at least once per year. For every meeting, an agenda shall be published online prior to the meeting and minutes of the meeting shall be provided in a timely manner following the meeting. Depending on the availability of funding, the members of the Board may be refunded travel costs for this meeting.
+#### 5.1.1 Appointed offices
 
 The Board assigns the following roles to individuals amongst its members:
 
 * one administrative chair
```

Comment:
The capitalization of "administrative chair" does not match that of "Technical Co-chairs" and "Elections Officers"

Reply:
Thanks for the catch. Capitalized throughout the document.